MoveCardToReview/ReadMe.md

```diff
@@ -14,13 +18,51 @@ Token to access GitHub API.
 
 ### `column-id`
 
-ID of the Kanban column where the card should be moved to. Typically ID of `Review in progress` column.
+ID of the Kanban column where the card should be moved to. Typically ID of `Review in progress` column. It is a number for Project classic and a string for V2. [This page](../docs/github.md) explains how this can be obtained.
+
+### `project-number`
+
+The project number when Projects V2 is used. [This page](../docs/github.md) explains how this can be obtained.
 
 ## Outputs
 
 None
 
-## Example usage
+## Usage examples
```

Comment:
Same as on the other PR - we'll need to add the section with prerequisites mentioning ticket to REs with example link (to the correct token)